app/views/incomeTaxHistory/IncomeTaxHistoryView.scala.html

```diff
@@ -37,11 +37,17 @@
 @(appConfig: ApplicationConfig, person: Person, incomeTaxYears: List[IncomeTaxYear])(implicit request: AuthenticatedRequest[_],
 messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
+@address = @{person.address.line1} @{person.address.line2} @{person.address.line3} @{person.address.postcode}
+
 @detailsContent = {
+
+@if(address.nonEmpty){
```

Comment:
would `@address.map { }` work instead of nonEmpty?

Reply:
@sergi-hmrc not sure how this would work??